installer/templates/phx_web/endpoint.ex

```diff
@@ -12,7 +12,9 @@ defmodule <%= endpoint_module %> do
 
   socket "/socket", <%= web_namespace %>.UserSocket,
     websocket: true,
-    longpoll: false
+    longpoll: false<%= if live do %>
+
+  socket "/live", Phoenix.LiveView.Socket, websocket: [connect_info: [session: @session_options]]<% end %>
```

Comment:
```suggestion
  socket "/live", Phoenix.LiveView.Socket,
    websocket: [connect_info: [session: @session_options]]<% end %>
```

Reply:
This is what I had at first, but it failed `--check-formatted` and that pesky formatter wants it on ea single line :)

Reply:
Bump the Elixir version. Since V1.7 or V1.8 we respect the user choice in here.